app/src/main/java/org/oppia/android/app/player/state/StateFragmentPresenter.kt

```diff
@@ -451,6 +464,14 @@ class StateFragmentPresenter @Inject constructor(
   /** Returns the checkpoint state for the current exploration. */
   fun getExplorationCheckpointState() = explorationCheckpointState
 
+  fun handleOnSavedInstance(): RawUserAnswer {
```

Comment:
The name doesn't quite match the return value--this ought to be tied to the specific value returned (the name currently reads as 'generic').